.prettierignore

```diff
@@ -2,4 +2,5 @@
 .type-coverage
 build
 /lib
-.npmrc
\ No newline at end of file
+.npmrc
+tests/fixtures/rules/component-tags-order
```

Comment:
How can I disable prettier for specific file?

Reply:
I often use `<!-- prettier-ignore -->`, but it is OK to use a `.prettierignore` file.

https://github.com/sveltejs/eslint-plugin-svelte/blob/e386e3417afa30d69e992f56dd8aff046a37c4cf/tests/fixtures/rules/indent/invalid/4-indent/test01-input.svelte#L1